examples/arithmetic/tests/bindings/test_arithmetic.py

```diff
@@ -1,6 +1,28 @@
 from arithmetic import *
+from arithmetic import ArithmeticError
```

Comment:
I don't know why the import star is not working 🤔

Reply:
In `uniffi_bindgen/src/bindings/python/templates/wrapper.py` at the bottom of the file, there's a blog that defines the special `__all__` attribute on the module. You'll need to add the error classes to that block in order to get them to appear as part of `import *`.

Reply:
Now it makes a lot more sense, thank you!